### DISCUSSION TITLE
access Meshcentral with nginx reverse proxy but in subdirectory?

### DISCUSSION THREAD

**Discussion by dryingwu** [Ylianst/MeshCentral#5028] [Q&A]
Hi, I set up meshcentral successfully and can access it via nginx reverse proxy if meshcentral is in the root of url (https://www.domainname.com/). I simply use the instruction in the user guide.

But I failed to move it to a subdirectory in url, like https://www.domainname.com/mesh/.

I can login but then with error saying **"Unable to connect web socket, click to reconnect**

Here is my config in nginx:

location **/mesh/** {

proxy_pass http://127.0.0.1:4430/;

proxy_http_version 1.1;

proxy_set_header Upgrade $http_upgrade;

proxy_set_header Connection "upgrade";

proxy_set_header Host $host;

proxy_set_header X-Forwarded-Host $host:$server_port;

proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;

proxy_set_header X-Forwarded-Proto $scheme;

}

Could anyone help me? Thanks.

**Comment by fornax-a** [comment]
My suggestion would be to go with subdomains, if you can.

**Comment by hnsay** [comment]
Have you managed to solve this?
